airbyte-integrations/connectors/source-facebook-marketing/source_facebook_marketing/spec.py

```diff
@@ -77,6 +77,14 @@ class Config:
         pattern=DATE_TIME_PATTERN,
         examples=["2017-01-26T00:00:00Z"],
     )
+    insights_lookback_window: Optional[PositiveInt] = Field(
+        title="Custom Insights Lookback Window",
+        description=(
+            "The attribution window"
+        ),
+        exclusiveMaximum=28,
+        default=28,
```

Comment:
default value = 28
maximum value is 27 (because 28 is exclusive)

as for me it's strange ?

Reply:
Can you also provide here `Minimum` value to be more explicit ?